sklearn/decomposition/tests/test_nmf.py

```diff
@@ -502,10 +503,10 @@ def test_nmf_decreasing():
             previous_loss = None
             for n_iter in range(1, 5):
                 W, H, n_iter = non_negative_factorization(
-                    X, W, H, beta_loss=beta_loss, n_components=n_components,
-                    max_iter=n_iter, alpha=alpha, solver=solver, tol=tol,
-                    l1_ratio=l1_ratio, verbose=0, regularization='both',
-                    random_state=0, update_H=True)
+                    X, W0.copy(), H0.copy(), beta_loss=beta_loss,
+                    n_components=n_components, max_iter=n_iter, alpha=alpha,
+                    solver=solver, tol=tol, l1_ratio=l1_ratio, verbose=0,
+                    regularization='both', random_state=0, update_H=True)
```

Comment:
I changed it according to your suggestion, changing to `n_iter=1` since we want only one more iteration starting from the previous results.

It made me realize I forgot `init='custom'` :worried:...